test/new-e2e/tests/agent-subcommands/secret/secret_nix_test.go

```diff
@@ -3,42 +3,49 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-package agentsubcommands
+package secret
 
 import (
 	"testing"
 
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
 	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
-
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2params"
 	"github.com/stretchr/testify/assert"
 )
 
-type agentSecretSuite struct {
-	e2e.Suite[e2e.AgentEnv]
+type linuxSecretSuite struct {
+	baseSecretSuite
 }
 
-func TestAgentSecretSuite(t *testing.T) {
-	e2e.Run(t, &agentSecretSuite{}, e2e.AgentStackDef())
+func TestLinuxSecretSuite(t *testing.T) {
+	t.Parallel()
+	e2e.Run(t, &linuxSecretSuite{}, e2e.AgentStackDef(e2e.WithVMParams(ec2params.WithOS(ec2os.UbuntuOS))), params.WithDevMode())
```

Comment:
Is `WithDevMode` supposed to be here? I thought it was only meant for developing tests, or is it used here to share state between the tests?

Reply:
You're right, I forgot to remove it mb 👍